modules/core/src/adapter/types/vertex-formats.ts

```diff
@@ -87,4 +89,8 @@ export type VertexFormat =
   | 'float32'
   | 'float32x2'
   | 'float32x3'
-  | 'float32x4';
+  | 'float32x4'
+  | 'float64'
+  | 'float64x2'
```

Comment:
I believe the register bank for vertex attributes is 16 bytes wide, i.e. there would only ever be `float64x2` if this was actually implemented. And if we want to emulate it with 2xfloat32 per float64 we would also face this limitation (i.e. there are no float32x6 or float32x8 types.)